content/docs/command-reference/exp/run.md

```diff
@@ -35,27 +35,24 @@ data and code updates, or <abbr>hyperparameter</abbr> tuning. You can use the
 `--set-param` option to change `dvc param` values on-the fly.
 
 Each `dvc exp run` creates a variation based on the latest project version
-committed to Git, and tracks this experiment automatically with an automatic
+committed to Git (`HEAD`), and tracks this experiment with an auto-generated
 name like `exp-bfe64` (which can be customized with the `--name` option).
 
 <details>
 
 ### How does DVC track experiments?
 
-Internally, `dvc exp` uses actual commits under custom
+Experiments are custom
 [Git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
-(found in `.git/refs/exps`). Each commit has the Git `HEAD` as parent and has
-it's own SHA-256 hash. These are not pushed to the Git remote by default (see
+(found in `.git/refs/exps`) with a commit whose parent is the `HEAD` at the time
+of running. Note that they're not pushed to the Git remote by default (see
 `dvc exp push`).
 
-> References have a unique signature similar to the
-> [entries in the run-cache](/doc/user-guide/project-structure/internal-files#run-cache).
-
 </details>
 
 The results of the last experiment can be seen in the <abbr>workspace</abbr>. To
 display and compare your experiments, use `dvc exp show` or `dvc exp diff`. Use
-`dvc exp apply` to roll back the workspace to a previous experiment
+`dvc exp apply` to roll back the workspace to a previous experiment.
```

Comment:
Maybe need a note here with a reminder that changes from `--queue/--temp` runs are not applied to the workspace.

Also, I think "roll back the workspace..." should maybe be something like "to apply the (git) changes from an experiment to the workspace". To me, using `roll back` sounds like we are reverting to an old changeset, which is not really the case.

Reply:
> changes from --queue/--temp runs are not applied to the workspace
> should maybe be something like "to apply the (git) changes from an experiment to the workspace

Updated in 0c7a060d.